### PR TITLE
Add similar locations example

### PR DESCRIPTION
**Pull request by diego-vicente**
Include an example for the Similar Locations functions in the CPG module, including a complete workflow:
- Creating a sample from the Iowa Liquor Stores dataset,
- Generating trade areas,
- Enriching the trade areas,
- Finding out similar locations.

It also includes a couple of maps to understand intermediate steps and the final result.

### CODE REVIEW

**Comment by diego-vicente** [comment]
I'll just leave here [a link to the example itself](https://cartodb-fb-documentation--pr518-feature-sc-278142-ad-9zjk03cz.web.app/analytics-toolbox-bigquery/examples/similar-locations-iowa/) (since the navigation does not seem to be working in the Firebase instance)

File: app/content/analytics-toolbox-bigquery/examples/similar-locations-iowa.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+date: "2022-12-21"
+categories:
+  - CPG
+aliases:
+  - /analytics-toolbox-bq/examples/similar-locations-iowa/
+---
+
+## Find similar locations based on their trade areas
+
+In the retail and CPG industry, it is common to find the need to understand a set of candidate locations when making different supply and stock decisions. In this example, we follow the steps that one can follow using CARTO and the Analytics Toolbox to rank a set of locations based on the demographic similarity to a chosen location.
```
Comment (xavipereztr) [inline app/content/analytics-toolbox-bigquery/examples/similar-locations-iowa.md:15]: In the retail and CPG industries

File: app/content/analytics-toolbox-bigquery/examples/similar-locations-iowa.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+We can visualize this sample in the following map:
+
+<iframe width=100% style="aspect-ratio: 1.5; margin-bottom: 2em" src="https://clausa.app.carto.com/map/96a10b79-4ed8-48fc-bee3-81e63752e20b"></iframe>
+
+Our sample has a column named `store_number` that uniquely identifies each of the locations. This column is relevant because it is a requirement for the `FIND_SIMILAR_LOCATIONS` function. We also filter those whose geographical location is known because we will use that location for the next step (generating the trade areas). Bear in mind that the Analytics Toolbox provides functions like [`GEOCODE_TABLE`](https://docs.carto.com/analytics-toolbox-bigquery/sql-reference/lds/#geocode_table) to infer the geography from an address, like in [this example](../geocoding-your-address-data).
+
+### Generating the trade areas
+
+In this step, we will define each location's trade area. We can understand these trade areas as the zones influenced by each of the store. The Analytics Toolbox also provides a handy function to achieve this, [`GENERATE_TRADE_AREAS`](https://docs.carto.com/analytics-toolbox-bigquery/sql-reference/cpg/#generate_trade_areas):
```
Comment (xavipereztr) [inline app/content/analytics-toolbox-bigquery/examples/similar-locations-iowa.md:52]: ...each of the stores.

**Review by xavipereztr — COMMENTED** [review]
Correct two typos and publish when we make the release

**Pull request merged**